sdk/storage/Azure.Storage.Files.DataLake/samples/Sample01a_HelloWorld.cs

```diff
@@ -0,0 +1,428 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection.Metadata.Ecma335;
+using Azure.Storage;
+using Azure.Storage.Files.DataLake;
+using Azure.Storage.Files.DataLake.Models;
+using NUnit.Framework;
+
+namespace Azure.Storage.Files.DataLake.Samples
+{
+    /// <summary>
+    /// Basic Azure DataLake Storage samples.
+    /// </summary>
+    public class Sample01a_HelloWorld : SampleTest
+    {
+        /// <summary>
+        /// Upload a file to a DataLake File.
+        /// </summary>
+        [Test]
+        public void Append()
+        {
+            // Create three temporary Lorem Ipsum files on disk that we can upload
+            int oneThirdPosition = SampleFileContent.Length / 3;
```

Comment:
Let's get a SampleFileContent with a length that is divisible by 3 to make the math simpler.